client/incus_storage_buckets.go

```diff
@@ -59,17 +59,13 @@ func(r *ProtocolIncus) GetStoragePoolBucketsWithFilter(poolName string, filters
 
 	buckets := []api.StorageBucket{}
 
-	v := url.Values{}
-	v.Set("recursion", "1")
-	v.Set("filter", parseFilters(filters))
-
```

Comment:
If this is a needed change, it should be in the previous (`client`) commit instead of here